docs/developer-docs/latest/developer-resources/database-apis-reference/entity-service/crud.md

```diff
@@ -0,0 +1,133 @@
+### Example
+
+```js
+const entries = await strapi.entityService.findMany('articles', {
+  fields: ['title', 'description'],
+  filters { title: 'Hello World' },
+  sort: { title: 'DESC' },
+  populate: { category: true },
+});
+```
+
+## create()
+
+Creates one entry and returns it
+
+Syntax: `create(uid: string, parameters: Params)` ⇒ `Entry`
+
+### Parameters
+
+| Parameter | Description                                | Type                           |
+| --------- | ------------------------------------------ | ------------------------------ |
+| `fields`    | Attributes to return           | `String[]`                     |
+| `populate`  | Relations, components and dynamic zones to [populate](/developer-docs/latest/developer-resources/database-apis-reference/entity-service/populating.md) | [`PopulateParameter`<Fa-Link color="grey"/>](/developer-docs/latest/developer-resources/database-apis-reference/entity-service/populating.md) |
+| `data`      | Input data                                 | `Object`                       |
+
+### Example
+
+```js
+const entry = await strapi.entityService.create('article', {
+  data: {
+    title: 'My Article',
+  },
+});
+```
+
+## update()
+
+Updates one entry and returns it.
+
+Syntax: `update(uid: string, id: ID, parameters: Params)` ⇒ `Entry`
+
+### Parameters
+
+| Parameter | Description                                | Type                           |
+| --------- | ------------------------------------------ | ------------------------------ |
+| `fields`    | Attributes to return           | `String[]`                     |
+| `populate`  | Relations, components and dynamic zones to [populate](/developer-docs/latest/developer-resources/database-apis-reference/entity-service/populating.md) | [`PopulateParameter`<Fa-Link color="grey"/>](/developer-docs/latest/developer-resources/database-apis-reference/entity-service/populating.md) |
+| `data`      | Input data                                 | `object`                       |
+
+### Example
+
+```js
+const entry = await strapi.entityService.update('article', 1, {
+  filters { id: 1 },
+  data: {
+    title: 'xxx',
+  },
+});
+```
+
+## delete()
+
+Deletes one entry and returns it.
+
+Syntax: `delete(uid: string, id: ID, parameters: Params)` ⇒ `Entry`
+
+### Parameters
+
+| Parameter | Description                                | Type                           |
+| --------- | ------------------------------------------ | ------------------------------ |
+| `fields`    | Attributes to return           | `String[]`                     |
+| `populate`  | Relations, components and dynamic zones to [populate](/developer-docs/latest/developer-resources/database-apis-reference/entity-service/populating.md) | [`PopulateParameter`<Fa-Link color="grey"/>](/developer-docs/latest/developer-resources/database-apis-reference/entity-service/populating.md) |
+
+### Example
+
+```js
+const entry = await strapi.entityService.delete('article', 1, {
+  filters { id: 1 },
```

Comment:
I don't think this is needed, the 3rd parameter in this honestly could just be left undefined since it's a delete